### PR TITLE
Port to 1.19

### LINKED ISSUE #3: Update Kontrolo for Minecraft 1.19
Another Minecraft release! As such, Kontrolo needs updating to the latest version.

### PR DESCRIPTION
**Pull request by Motschen**
Signed-off-by: Motschen <motschen@midnightdust.eu>
All of the code changes were done completely by myself.
Sorry for having to close the last PR, I'm using GitHub Desktop and couldn't find out how to rebase the commit.

### CODE REVIEW

**Comment by Lucxjo** [comment]
This PR deals with issue #3

File: gradle.properties [hunk 1]
```diff
@@ -2,20 +2,21 @@ kotlin.code.style=official
...
+yarn_mappings=1.19+build.4
+loader_version=0.14.8
 #Fabric api
-fabric_version=0.43.1+1.18
-loom_version=0.10-SNAPSHOT
+fabric_version=0.57.0+1.19
+loom_version=0.12-SNAPSHOT
 # Mod Properties
-modVersion=1.5.1
+modVersion=1.6.0
```
Comment (Lucxjo) [inline gradle.properties]: This should still be 1.5.1, no changes have been made other than a simple port so no version bump is needed.

**Review by Lucxjo — CHANGES_REQUESTED** [review]
All changes look good. The version of the mod bundle may need changing back but I'll do this later once I've checked a couple of things. Mod needs to be confirmed to run by 2022-07-10 (that's a me job). Hopeful release of 1.19 version by 2022-07-10.

Thanks for your contributions @Motschen!

File: gradle.properties (hunk 1 above)
Comment (Motschen) [reply gradle.properties]: Done in the latest commit.

**Review by Lucxjo — APPROVED** [review]

**Comment by Lucxjo** [comment]
Changed base branch to develop. I need to change a couple of things before release.

**Comment by Lucxjo** [comment]
all-contributors please add @Motschen for code

**Comment by Lucxjo** [comment]
@all-contributors please add @Motschen for code

**Pull request merged**